elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/PlayerStatsWithView.java

```diff
@@ -200,8 +200,7 @@ public void setSubCountryIsoCode(String isoCode) {
         this.subCountryIsoCode = isoCode;
     }
 
-    @ToOne
-    @JoinTo(joinClause = "%from.country_id = %join.id")
+    @Join("%from.country_id = %join.id")
```

Comment:
I'm worried countryView is going to show up in the GraphQL schema as an attribute.  While we can always have the developer add `@Exclude`, it would be better if JoinTo automatically excluded the property.

One way to do this is to promote AggregationStore annotations to elide-annotations as  first class citizens.  I think Elide 5.0 should collapse elide-annotations and elide-core because there are natural dependencies between them.
I don't want to do that collapse now though because it will make rebasing harder.

Another option is to allow the entity dictionary to accept hints when the data store binds the entities.  We could make another bindEntity method that takes additional annotations to exclude while binding.  I think for now, I favor this option.

Reply:
The field is hidden in both JSON API and GraphQL. I have this latest commit.
https://github.com/yahoo/elide/pull/1185/commits/3053cdf49c4b019a7204f1418e77678efe8eac40

Reply:
I don't want to add a new annotation for this.  We already have Exclude.   I think Join should be implicitly hidden (without having to add another declaration).